### PR TITLE
Pylondinium 2018

### PR DESCRIPTION
**Pull request by pganssle**
This adds the PyLondinium 2018 conference, mostly using the provided Youtube tool with a small amount of cleanup using an ad hoc script that extracts data from the video titles.

Most videos still need the full names of the speakers, the "recorded" parameter and the duration of the videos. Some probably need descriptions. A decent amount of this information can probably be extracted from [the PyLondinium website](https://github.com/PyLondinium/pylondinium.github.io), if videos and talk IDs can be associated easily.

CC @mariocj89 @pablogsal

### CODE REVIEW

File: pylondinium-2018/category.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "title": "PyLondinium"
```
Comment (redapple) [inline pylondinium-2018/category.json]: You'll need the year here

Comment (pganssle) [reply pylondinium-2018/category.json]: Done, thanks.

**Comment by redapple** [comment]
You should use the `"language"` key instead of `"lang"`.
Also, could you run `tools/reserialize.py`?

**Comment by pganssle** [comment]
@redapple 

> You should use the "language" key instead of "lang".
> Also, could you run tools/reserialize.py?

What was this supposed to do? I got no changes from:

```
python tools/reserialize.py --all pylondinium-2018
```

**Comment by pganssle** [comment]
I've switched all `lang` keys over to `language`.

**Comment by redapple** [comment]
@pganssle , it reformats the JSON files so that they are consistent (sorted, 2 spaces etc.);
This is what I usually run at the root of the repo:
`python tools/reserialize.py -a .`

I've pushed to your branch directly.

**Comment by redapple** [comment]
Thanks @pganssle for the contribution!

**Comment by pganssle** [comment]
@redapple Very strange, `reserialize` did nothing for me, I wonder why.

**Pull request merged**